container/root/etc/nginx/nginx.conf

```diff
@@ -19,6 +19,7 @@ error_log /dev/stdout warn;
 # Number of file descriptors used for nginx
 worker_rlimit_nofile 40000;
 
+#plugins_placeholder
```

Comment:
if this placeholder is specific for brotli placeholders, you might want to make the name more specific, or in the routine where you are replacing this placeholder don't replace it but instead add to it, that way other init scripts can add more plugins

Reply:
@ingluisjimenez sounds good to me, let me fix it!